src/components/Header/LanguageSelect.tsx

```diff
@@ -27,11 +27,13 @@ const LanguageSelect: FunctionComponent<{ lang: string }> = ({ lang }) => {
 				className="header-button language-select"
 				value={lang}
 				aria-label="Select language"
-				onChange={(e) => {
-					const newLang = e.target.value;
-					const [_leadingSlash, _oldLang, ...rest] = window.location.pathname.split('/');
-					const slug = rest.join('/');
-					window.location.pathname = `/${newLang}/${slug}`;
+				onChange={(e: ChangeEvent) => {
+					if (e.target instanceof HTMLSelectElement) {
+						const newLang = e.target.value;
+						const [_leadingSlash, _oldLang, ...rest] = window.location.pathname.split('/');
+						const slug = rest.join('/');
+						window.location.pathname = `/${newLang}/${slug}`;
+					}
 				}}
```

Comment:
I think we can probably skip the runtime `if` by using `currentTarget`:

```suggestion
				onChange={(e: ChangeEvent<HTMLSelectElement>) => {
					const newLang = e.currentTarget.value;
					const [_leadingSlash, _oldLang, ...rest] = window.location.pathname.split('/');
					const slug = rest.join('/');
					window.location.pathname = `/${newLang}/${slug}`;
				}}
```